mowgli_etl/pipeline/wdc/data_exploration/buckets_to_dict.py

```diff
@@ -38,22 +38,29 @@
 for key in bucketer.averages.keys():
     if not bucketer.averages[key].bucket:
         continue
-    buckets[f"Bucket{bucketer.averages[key].bucket}"].append((key,bucketer.averages[key].volume))
+    buckets[f"Bucket{bucketer.averages[key].bucket}"].append(
+        (key, bucketer.averages[key].volume)
+    )
 
 for key in buckets.keys():
     print(f"{key} contains {len(buckets[key])} products")
 
 max_length = max([len(item) for item in buckets.values()])
 for key in buckets.keys():
     while len(buckets[key]) < max_length:
-        buckets[key].append(('',''))
+        buckets[key].append(("", ""))
```

Comment:
I'm not sure what this is doing.

Usual thing is to use | to separate values in a comma-separated column e.g.,
```
x|1,y|2
```